### ISSUE TITLE
Add crate parameter

### ISSUE THREAD

**Issue by stepancheg** [palfrey/serial_test#123]
```
#[serial_test(crate = wrapper::__derive_refs::serial)]
```

This is for proc-macros generating `#[serial_test]` annotations: they should rely on reexports of `serial` crate.

**Comment by stepancheg** [comment]
This is how serde is doing it for example: https://serde.rs/container-attrs.html#crate

Use case is this: I'm developing a macro like `#[my_test_utils::my_test]` in a crate `my_test_utils`. It should expand to:

```
#[serial_test::serial]
#[test]
fn xxx() {
  // custom my test logic...
  ...
}
```

However, `serial_test` may not be imported where this `my_test_utils` crate is imported.

`#[my_test]` could be expanded instead into

```
#[my_test_utils::__derive_refs::serial_test::serial(crate = my_test_utils::__derive_refs::serial_test)]
#[test]
fn xxx() {
  // custom init...
  ...
}
```

so that when `#[serial]` ireferences `serial_test` crate function, it references them as `my_test_utils::__derive_refs::serial_test::xxx` instead of `serial_test::xxx`.

Please let me know, if this explanation does not help.

**Comment by palfrey** [comment]
Yeah, it makes more sense now. I'm busy with a whole bunch of things at the moment, and won't get to this anytime soon. OTOH, if you're able to provide a PR for this, I'm happy to review.

**Comment by palfrey** [comment]
https://github.com/palfrey/serial_test/pull/130 should fix this. Can you check against your use case @stepancheg?
